### PR TITLE
bugfix: visibility consistency is not using local qurom

### PR DESCRIPTION
**Pull request by wxing1292**

### CODE REVIEW

File: common/persistence/cassandraVisibilityPersistence.go [hunk 1]
```diff
@@ -230,7 +229,7 @@ func (v *cassandraVisibilityPersistence) ListOpenWorkflowExecutions(
 		request.DomainUUID,
 		domainPartition,
 		common.UnixNanoToCQLTimestamp(request.EarliestStartTime),
-		common.UnixNanoToCQLTimestamp(request.LatestStartTime)).Consistency(v.lowConslevel)
+		common.UnixNanoToCQLTimestamp(request.LatestStartTime))
```
Comment (samarabbas) [inline common/persistence/cassandraVisibilityPersistence.go]: Actually thinking about this, we should keep low consistency level for read queries on visibility for performance reasons.  We provide eventual consistency for visibility store.  So we should keep the low consistency read for performance reasons.

Comment (wxing1292) [reply common/persistence/cassandraVisibilityPersistence.go]: agree that we should use less Cassandra capacity, however, the canary test for visibility can (and it actually did) experience issue, i.e. workflow started but list open workflow return 0.

Comment (wxing1292) [reply common/persistence/cassandraVisibilityPersistence.go]: So, overall, it depends on what kind of SLA that we guarantee.

**Comment by wxing1292** [comment]
The conclusion is visibility part only provide eventually consistency, close PR

**Pull request closed without merging**